tests/unittest/auto_deploy/singlegpu/models/test_openelm_modeling.py

```
@@ -470,3 +470,133 @@ def test_openelm_model_can_be_exported():
+    # `use_cache=True` would crash Apple's remote config under transformers 5.x;
+    # the vendored class must accept it without raising.
+    config = OpenELMConfig(use_cache=True, **_OPENELM_270M)
+
+    assert config.model_type == "openelm"
+    assert config.num_query_heads == _EXPECTED_NUM_QUERY_HEADS
+    assert config.num_kv_heads == _EXPECTED_NUM_KV_HEADS
+    assert len(config.ffn_multipliers) == 16
+    # per-layer FFN multipliers are a linspace(0.5, 4.0, 16)
+    assert config.ffn_multipliers[0] == 0.5 and config.ffn_multipliers[-1] == 4.0
+    # GQA divisibility holds for every layer
+    assert all(q % k == 0 for q, k in zip(config.num_query_heads, config.num_kv_heads))
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)